doc/source/whatsnew/v0.20.0.txt

```diff
@@ -430,7 +430,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
     import os
     os.remove('styled.xlsx')
 
-See the :ref:`Style documentation <style>` for more detail.
+See the :ref:`Style documentation <style.ipynb>` for more detail.
```

Comment:
actually would be better for this one to go directly to: http://pandas-docs.github.io/pandas-docs-travis/style.html#Export-to-Excel